deploy.json.tmpl

```diff
@@ -0,0 +1,50 @@
+{
+  "id": "${CS_MARATHON_APP_ID}",
+  "cmd": null,
+  "cpus": 0.2,
+  "mem": 512,
```

Comment:
gddo is stateless, right? could we drop this memory figure way down and save some cluster resources? say 128?